airbyte-cdk/python/pyproject.toml

```diff
@@ -62,6 +62,7 @@ unstructured = { version = "0.10.27", extras = ["docx", "pptx"], optional = true
 pyjwt = "^2.8.0"
 cryptography = "^42.0.5"
 pytz = "2024.1"
+datamodel-code-generator = "0.11.19"
```

Comment:
why is the dependency moved from the dev group?

Reply:
deleted